pkgs/applications/office/kmymoney/default.nix

```diff
@@ -22,5 +31,6 @@ stdenv.mkDerivation rec {
     homepage = http://kmymoney2.sourceforge.net/;
     description = "KDE personal money manager";
     inherit (kdelibs.meta) platforms maintainers;
+    priority = 100;
```

Comment:
@ttuegel I think this is because of conflicts with a file. Something about mime. I know that having skrooge and another kde5 application gave me a problem and I had to set the priority (using nix-env)